src/components/expansion/ExpansionSetRuleSelection.svelte

```diff
@@ -0,0 +1,15 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  export let activityName: string;
+  export let expansionRules: ExpansionRule[];
+  export let selectedExpansionRules: Record<string, number> = {};
+
```

Comment:
Remove newline

Reply:
you don't think function variables should be separated?

Reply:
In this case since it's a one-liner I think it's okay. Looks cleaner :)